brave/brave-ios
# Fix #8379: Transaction Details v2 UI

## Summary of Changes
- Updated UI for Transaction Details

This pull request fixes #8379

## Submitter Checklist:


## Test Plan:
1. Make a few different types of transactions; they do not need to be confirmed (though should confirm at least 1 to verify transaction status states).
2. Open Activity tab and tap on any transaction to open transaction details.
3. Verify From/To/Transaction Fee rows appear below transaction details. 
4. Verify From/To fields copy button copies the respective address
5. Confirm at least 1 transaction.
6. Open transaction details for confirmed transaction.
7. Verify Transaction Hash row appears. Verify copy button copies tx hash, and link button opens the explorer URL.
8. Open accounts detail for an account you made a transaction on, verify transactions open using new Transaction Details UI.
9. Open asset detail for an asset you made a transaction with, verify transactions open using new Transaction Details


## Screenshots:

![Send NFT - light](https://github.com/brave/brave-ios/assets/5314553/0ad1786c-5818-46dc-958d-b6a2673d47df) | ![Send NFT - dark](https://github.com/brave/brave-ios/assets/5314553/b7fa103f-3b3e-4a14-9bec-79d30680d6be) | ![Send ETH - light](https://github.com/brave/brave-ios/assets/5314553/111cabd8-abf1-4782-b4b5-67045d2f9625) | ![Send ETH - dark](https://github.com/brave/brave-ios/assets/5314553/be03ec5d-0193-46c8-b320-ab0acc8ea22b)
--|--|--|--
![Approve Value - light](https://github.com/brave/brave-ios/assets/5314553/6c7c3638-4941-44fc-9fc8-f174f6f2b039) | ![Approve Value - dark](https://github.com/brave/brave-ios/assets/5314553/c53ce51f-a82b-4a4c-9dae-416d04e1591e) | ![Approve Unlimited - light](https://github.com/brave/brave-ios/assets/5314553/723a7ceb-181a-4846-8169-ca27cafe2481) | ![Approve Unlimited - dark](https://github.com/brave/brave-ios/assets/5314553/f77cc1dc-1d82-4c3c-bc5a-534da8aaa9ab)
![Swap - light](https://github.com/brave/brave-ios/assets/5314553/fe54047e-925f-4fa0-9e1c-3c983e6b7a9f) | ![Swap - dark](https://github.com/brave/brave-ios/assets/5314553/0242f11a-1100-49b3-959b-7baaa5c84845) | ![Solana Swap - light](https://github.com/brave/brave-ios/assets/5314553/0af95a30-3901-430e-965a-958624d65612) | ![Solana Swap - dark](https://github.com/brave/brave-ios/assets/5314553/e14ea395-7784-4d52-906a-09eb1d697b70)


## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
